test/conftest.py

```diff
@@ -36,9 +36,11 @@
 def pytest_addoption(parser):
     parser.addoption('--docker-base-name', default='preprod-mxnet')
     parser.addoption('--region', default='us-west-2')
-    parser.addoption('--framework-version', required=True)
-    parser.addoption('--py-version', required=True, choices=['2', '3'])
-    parser.addoption('--processor', required=True, choices=['gpu', 'cpu'])
+    parser.addoption('--framework-version', default='1.2.1')
+    parser.addoption('--py-version', default='3', choices=['2', '3'])
+    parser.addoption('--processor', default='cpu', choices=['gpu', 'cpu'])
+    parser.addoption('--account-id', default='142577830533')
```

Comment:
is it ok to expose our dev account id here?

Reply:
probably not - we'll have to remove it from [your code](https://github.com/aws/sagemaker-tensorflow-container/blob/script-mode/test/conftest.py#L39) too then

Reply:
yeah. i will remove mine too.